### ISSUE TITLE
Default series description not describing all series in a Stock chart.

### ISSUE THREAD

**Issue by marvin19** [highcharts/highcharts#20656]
#### Expected behaviour
I expect all three series to have a default series description.

#### Actual behaviour
`aria-label` is missing for the first two series. When navigating with screen reader, skips first two series and goes to last series and describes "Navigator 1, series 3 of 3.... ". What seems to be helping is clicking one of the range selector buttons, and then clicking "All"-button. Description is then working for the other two series. Could potentially be something with async data-loading and not related to Stock since it is working after clicking one of the range selector buttons? 

#### Live demo with steps to reproduce
https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/stock/demo/stock-tools-gui

#### Product version
Highcharts Stock

#### Affected browser(s)
Chrome, Safari with VoiceOver from MacOS Sonoma 14.2.1
